test/commands/scorecard.spec.js

```diff
@@ -0,0 +1,77 @@
+// Import Node.js Dependencies
+import { fileURLToPath } from "node:url";
+import path from "node:path";
+
+// Import Third-party Dependencies
+import test from "tape";
+
+// Import Internal Dependencies
+import { initCliRunner } from "../helpers/cliCommandRunner.js";
+import { getExpectedScorecardLines } from "../helpers/utils.js";
+import { getCurrentRepository } from "../../src/commands/scorecard.js";
+
+// CONSTANTS
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const kProcessDir = path.join(__dirname, "..", "process");
+const kProcessPath = path.join(kProcessDir, "scorecard.js");
+const kOpenSSFScorecardRestApi = "https://api.securityscorecards.dev";
```

Comment:
Should we export this as CONSTANT from the SDK package?

Reply:
Yes I think its a good idea